src/ofxAudioAnalyzerAlgorithms.cpp

```diff
@@ -58,28 +58,35 @@ float ofxAABaseAlgorithm::getValueNormalized(float min, float max, bool doClamp)
 float ofxAABaseAlgorithm::getValueDbNormalized(float min, float max, bool doClamp){
     return ofMap(getValueDb(), min, max, 0.0, 1.0, doClamp);
 }
+
+//----------------------------------------------
+float ofxAABaseAlgorithm::smoothAttackRelease(float currentValue, float newValue, float smoothAttck, float smoothRlse) {
```

Comment:
The idea of smoothing is to average the results to avoid high freq oscillations in the analysis. The problem with this is that it may delay a bit the response of the analysis if the smoothing is too high the analysis starts. I understand that this is what you're trying to solve with this change. At first sight I see one main problems with your proposal: 
The concept of attack / release only would apply to RMS/Energy analysis, which I think are the ones you're using. But it's not useful for pitch changes for example... Why would you use different kind of smth values when pitch goes down or up?
For this reason I don't think it would be the best to add it to the common getValue functionality. We could add Util function to smooth an specific value differently perhaps. What do you think?

Reply:
Also, why would you need to have a different smooth when values go up and a different one when they go down? Why not using a low smth value for all?